src/commands/deploy.ts

```diff
@@ -12,16 +13,28 @@ export default class Deploy extends CustomCommand {
   static flags = {
     ...CustomCommand.globalFlags,
     yes: oclifFlags.boolean({ char: 'y', description: 'Answer yes to confirmation prompts' }),
+    force: oclifFlags.boolean({ char: 'f', description: 'Force deploy' }),
   }
 
   static args = [{ name: 'appId' }]
-
   async run() {
-    const {
-      args: { appId },
-      flags: { yes },
-    } = this.parse(Deploy)
-
-    await appsDeploy(appId, { yes })
+    const configClient = ToolbeltConfig.createClient()
+    const { featureFlags } = await configClient.getGlobalConfig()
```

Comment:
I'm not a specialist in Typescript but I had often see this type of declaration in the beginning of the function. WDYT?

Reply:
Never mind, I think it's impossible to do what I was thinking.